@AH-64D Apache Longbow/Addons/fza_ah64_mpd/config.cpp

```diff
@@ -0,0 +1,29 @@
+class CfgPatches
+{
+	class fza_ah64_mpd
+	{
+        units[] = {};
+		author = "Franze, Nodunit, Sacha 'Voodooflies' Oropeza, Keplager & Community";
```

Comment:
needs updating to match control author list
required version needs updating
required addon needs updating